[Bug]: pre-commit checks are failing for protobuf generated files

### Describe the bug

Protobuf is an industry standard format for publishing and receiving messages.  CAP uses these messages in the message bus. 
 protoc generates files that bind the message to python. These files are not meant to be fixed by hand.  

There is a pre-commit ruff test that fails.  This test seems to fail on the following line:

if _descriptor._USE_C_DESCRIPTORS == False:

The recommendation is to change it to:

if not _descriptor._USE_C_DESCRIPTORS:

------------------------------------- Details below -------------------------------------

ruff.....................................................................Failed
- hook id: ruff
- exit code: 1

samples/apps/cap/py/autogencap/proto/Autogen_pb2.py:23:4: E712 Avoid equality comparisons to `False`; use `if not _descriptor._USE_C_DESCRIPTORS:` for false checks
samples/apps/cap/py/autogencap/proto/CAP_pb2.py:23:4: E712 Avoid equality comparisons to `False`; use `if not _descriptor._USE_C_DESCRIPTORS:` for false checks
Found 2 errors.

### Steps to reproduce

Create a PR with:

if some_condition == False:

### Model Used

NA

### Expected Behavior

Code formatting / pre-commit-check should not be fail.



### Screenshots and logs

https://github.com/microsoft/autogen/actions/runs/8468895590/job/23202947149?pr=2189#step:6:117

### Additional Information

_No response_

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -52,6 +52,7 @@ exclude = [
      # This file needs to be either upgraded or removed and therefore should be
     # ignore from type checking for now
     "math_utils\\.py$",
+    "samples\\apps\\cap\\py\\autogencap\\proto\\.*\\.py",
 ]
 ignore-init-module-imports = true
 unfixable = ["F401"]

